src/kirby/components/list/list-item/list-item.component.html

```diff
@@ -1,6 +1,7 @@
-    <div class="title-container">
-        <div class="title">{{title}}</div>
-        <div class="subTitle">{{subTitle}}</div>
-    </div>
-    <div class="details-container">{{detail}}</div>
-    <div class="amount-container">{{amount}}</div>
+<div class="title-container">
```

Comment:
Good to see som boy scout work done 👍

Reply:
True - on the other hand, this file has nothing to do with issue/PR so the change pollutes the version control history...

Reply:
I think it is fair to do a little boy scout work/refactoring without it is polluting the commit too much. There is a balance.